tests/nested/manual/fakeinstaller/task.yaml

```diff
@@ -172,6 +171,12 @@ execute: |
   remote.exec "snap changes" | MATCH "Done.* Initialize system state"
   remote.exec "snap list" | MATCH pc-kernel
 
+  # Check for the generic serial assertion
+  retry -n 100 sh -c 'remote.exec "snap changes" | MATCH "Done.*Initialize device"'
+  remote.exec "snap model --serial --assertion" | MATCH "authority-id: generic"
+  remote.exec "snap model --serial --assertion" | MATCH "brand-id: generic"
+  remote.exec "snap model --serial --assertion" | MATCH "model: generic-classic"
```

Comment:
these two (brand-id, model) should be the values from the model used, not the ones for the generic assertion shipped with snapd

Reply:
Fixed